docs/admin/kube-apiserver.md

```diff
@@ -18,6 +18,11 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
+<!-- TAG RELEASE_LINK, added by the munger automatically -->
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.2/docs/admin/kube-apiserver.md).
```

Comment:
versionize-docs.sh runs update-generated-docs.sh which runs update-munge.docs.sh which is going to then insert these RELEASE_LINKs into the docs.